src/plugins/intel_gna/samples/transformer/src/app/menu.cpp

```diff
@@ -0,0 +1,173 @@
+#include <openvino/pass/serialize.hpp>
+#include <string>
+
+#include "configuration/transformer_configuration_loader_impl.hpp"
+#include "logger/logger.hpp"
+#include "passes/app_pass_manager.hpp"
+#include "transformer/model_transformer_app.hpp"
+#include "transformer/model_transformer_gna.hpp"
+
+using namespace transformation_sample;
+
+static const std::string help_message = "Print helper message.";
+DEFINE_bool(h, false, help_message.c_str());
+
+std::string config_message = "Path to .json file with transformer application configuration.";
+DEFINE_string(config, "", config_message.c_str());
+
+std::string transformation_message = "Name of application defined transformation to be applied on input_model.";
+DEFINE_string(transformation, "", transformation_message.c_str());
+
+std::string input_model_message = "Path to .xml file with input model in IR format.";
+DEFINE_string(input_model, "", input_model_message.c_str());
+
+std::string output_model_message = "Path to file to export transformend model with .xml extension.";
+DEFINE_string(output_model, "", output_model_message.c_str());
+
+static const std::string list_app_transformations_message = "List application defined transformations.";
+DEFINE_bool(list_app_transformations, false, list_app_transformations_message.c_str());
+
+const std::string Menu::kWeightsFileExtension = ".bin";
+constexpr char Menu::kFileNameDelimiter;
+
+const std::string example_command_config =
+    "-input_model model.xml -output_model transformed_model.xml -config configuration.json";
+const std::string example_command_transformation =
+    "-input_model model.xml -output_model transformed_model.xml -transformations transformation_name";
+
+Menu::Menu(int argc, char* argv[]) {
+    init(argc, argv);
+}
+
+static void print_usage() {
+    log_info() << std::endl;
+    log_info() << "gna_transformer_app [OPTION]" << std::endl;
+    log_info() << "Options:" << std::endl;
+    log_info() << std::endl;
+    log_info() << "    -h                            " << help_message << std::endl;
+    log_info() << "    -input_model \"<path>\"       " << input_model_message << std::endl;
+    log_info() << "    -output_model \"<path>\"      " << output_model_message << std::endl;
+    log_info() << "    -config \"<path>\"            " << config_message << std::endl;
+    log_info() << "    -transformation \"<string>\"  " << transformation_message << std::endl;
+    log_info() << "    -list_app_transformations     " << list_app_transformations_message << std::endl;
+    log_info() << std::endl;
+    log_info() << "    commands examples:" << transformation_message << std::endl;
+    log_info() << std::endl;
+
+    log_info() << example_command_config << std::endl;
+    log_info() << example_command_transformation << std::endl;
+}
+
+void Menu::execute_action() {
+    if (m_action) {
+        m_action();
+        log_info() << "Transformation successed" << std::endl;
+    }
+}
+
+void Menu::init(int argc, char* argv[]) {
```

Comment:
it looks like some dependency to frontend is missing their are not build if only gna_transformer_app is build - this cause to the issue with loading model.